mlperf_logging/rcp_checker/training_1.1.0/rcps_dlrm.json

```diff
@@ -3,13 +3,15 @@
   "dlrm_ref_32768":
   {
     "Benchmark": "dlrm",
+    "Creator": "NVidia",
+    "When": "Prior to 1.0 submission",
+    "Platform": "TBD",
     "BS": 32768,
     "Hyperparams": {
        "opt_base_learning_rate": 18,
        "opt_learning_rate_warmup_steps": 8000,
        "lr_decay_start_steps": 70000,
        "sgd_opt_base_learning_rate": 18,
-       "sgd_opt_learning_rate_decay_poly_power": 2,
```

Comment:
Is removing `sgd_opt_learning_rate_decay_poly_power` discussed in the training WG? I'm unaware of this.

Reply:
This is tied to reference, so there is no point keep that in the file. The training WG is aware of that.